src/twisted/internet/test/test_unix.py

```diff
@@ -382,6 +383,62 @@ def test_fileDescriptorOverrun(self):
         test_fileDescriptorOverrun.skip = sendmsgSkip
 
 
+    def test_multiFileDescriptorReceivedPerRecvmsg(self):
+        """
+        Verify that _SendmsgMixin handles receiving multiple file descriptors
```

Comment:
As per https://jml.io/pages/test-docstrings.html we should remove "verify that" from the docstring.